src/modules/app/AppContainer.js

```diff
@@ -1,15 +1,15 @@
 import 'react-select/dist/react-select.css';
 import { connect } from 'react-redux';
 import { routerActions } from 'react-router-redux';
-import { TYPES as PICK_TYPES, clearNodes, clearObjects } from '../../reducers/picklist';
-import { loadBuildinfo } from '../../reducers/auth';
-import { clearSearch } from '../../reducers/objectsearch/actions';
-import App from '../../components/app';
+import { TYPES as PICK_TYPES, clearNodes, clearObjects } from '../picklist/picklistReducer';
+import { loadBuildinfo } from './appReducer';
+import { clearSearch } from '../objectsearch/actions';
```

Comment:
jeg har egentlig prøvd å endre minst mulig på selve innholdet i filene. objectsearch har lenge hatt 3 file:  actions, reducer og constants. De andre har enten 1 reducer ala picklistReducer.js for eks.. eller flere reducer ala storagefacility .. men dette er slik det er i dag .. og min plan med refaktoreringen er kun å flytte filer sammen som bør være sammen .. ved å flytte de sammen, så øker det muligheten til å kunne refaktorere "de indre deler" av en modul, fordi alt ligger på et sted, og fordi vi på grunn av at jeg har fjernet index.php fra overalt kan se hvor ting er brukt